app/uk/gov/hmrc/rasapi/config/Filters.scala

```diff
@@ -20,18 +20,17 @@ import akka.stream.Materializer
 import com.google.inject.{Inject, Singleton}
 import play.api.http.DefaultHttpFilters
 import play.api.mvc.{Filter, RequestHeader, Result}
-import uk.gov.hmrc.play.bootstrap.filters.MicroserviceFilters
+import uk.gov.hmrc.play.bootstrap.backend.filters.BackendFilters
 
 import scala.concurrent.{ExecutionContext, Future}
 
 class XSSProtectionFilter @Inject() (val mat: Materializer)(implicit ec: ExecutionContext) extends Filter {
-
   override def apply(f: RequestHeader ⇒ Future[Result])(rh: RequestHeader): Future[Result] =
     f(rh).map(_.withHeaders("X-XSS-Protection" → "1; mode=block", "X-Content-Type-Options" → "nosniff"))
 
 }
 
 @Singleton
-class Filters @Inject() ( defaultFilters : MicroserviceFilters,
-                          xssProtectionFilter: XSSProtectionFilter
+class Filters @Inject() (defaultFilters : BackendFilters,
```

Comment:
I'm not sure if these filters are needed still, since they seem to just be doing the default stuff? (plus, BackendFilters shows up as deprecated)

Reply:
BackendFilters is brought in as part of backend.conf, however since that filter is being overwritten with this one, it needs to be re-brought in  with the extra filters. (I think).

Reply:
I've just had a quick gander: those filters seem to default to the values explicitly set in the XSSProtectionFilter anyway (under `play.filters.headers.xssProtection` and `play.filters.headers.contentTypeOptions`)

Reply:
Oh interesting, the custom filters aren't needed then.